src/script/api/script_list.cpp

```diff
@@ -15,6 +15,9 @@
 
 #include "../../safeguards.h"
 
+/** Number of bytes per item to charge to script allocation limit. */
+static const size_t SCRIPT_LIST_BYTES_PER_ITEM = 64;
```

Comment:
```suggestion
static const size_t SCRIPT_LIST_BYTES_PER_ITEM = 96;
```
Calculated with the help of visual studio.

<img width="217" height="20" alt="Image" src="https://github.com/user-attachments/assets/513d51d2-8fb8-4bf2-9c50-832441b7e112" />

<img width="1560" height="252" alt="Image" src="https://github.com/user-attachments/assets/63583b9a-62c7-42e1-8655-7a91cb9d9357" />

2 containers, 18960632 * 2 = 37921264 allocations
1820220672 / 18960632 = 96 bytes per item